core/websocket.go

```diff
@@ -0,0 +1,46 @@
+package core
+
+import (
+	"golang.org/x/net/websocket"
+)
+
+func queryWebSocket(endpoint *Endpoint, result *Result) {
+	const (
+		Origin             = "http://localhost/"
+		MaximumMessageSize = 1024 // in bytes
+	)
```

Comment:
I'm a bit conflicted here, because historically, actual calls were made from the `client` package here: https://github.com/TwiN/gatus/blob/master/client/client.go as opposed to the core package, but if you used the dns file as an example, I can see why you'd put it in the core package instead.

Still, could you move this functionality in the client package instead? I'll take care of the dns one separately to avoid future confusion

Reply:
Yeah, I used the DNS as example.

I can move the code to the `client` package, no problem. Will move the function there and follow the same terminology. I should rename the function to `CanCreateWebSocketConnection`?

Reply:
Also, do you want me to rebase this branch on top of current `master`? I saw that you merged `master` in here.

Reply:
Naming it `QueryWebsocket` is fine; you'll have to modify the function's signature so that it doesn't pass `core.Endpoint` and `core.Result` though, just like the other functions in the `client` package.

As for the rebase question- I'll squash when I merge anyways, so don't worry about it.

Reply:
I moved the function to the client package. Had to add other arguments and return to accomododate a `body` and a return message. Hope it is fine, but if you know of a better way I can rework it.

And a noob question: how do I compile this branch?
```shell
$ go build -a -installsuffix cgo -o gatus .
# github.com/TwiN/gatus/v5/core
core/endpoint.go:347:47: undefined: client.queryWebSocket
```

Reply:
Oh, I needed to have the function in Uppercase.